### ISSUE TITLE
Current version of `marked` creates npm audit issue

### ISSUE THREAD

**Issue by joehuanguf** [storybookjs/marksy#78]
npm is giving me a security issue with the current version of the `marked` dependency. It recommends that we upgrade to `0.7.0`.

<img width="573" alt="Screen Shot 2019-07-25 at 1 27 14 PM" src="https://user-images.githubusercontent.com/51337028/61906313-f271e000-aedf-11e9-9f4f-a01e56a5ea5a.png">

**Comment by fabb** [comment]
Related: https://github.com/storybookjs/marksy/commit/2e7f73f59d3ef9ede8a4b1888c1c05fea0d20566

**Comment by lukemarsh** [comment]
@fabb I see that `marked` has been downgraded but the issue has been fixed in the `0.7.0` patch

**Comment by fabb** [comment]
Yes. When `marked` is upgraded, the linked issue with sanitized inline elements will need fixing.

**Comment by XGHeaven** [comment]
@fabb I found that `marked` has been downgraded at version `8.0.0`. But `@storybook/addon-info` still use `^7.0.0`. It's also not works. 😭

**Comment by ranand** [comment]
`marked` is at `0.8.0` now. is it possible to upgrade `marked` version in package.json?

**Comment by TheresaBeckerLR** [comment]
> `marked` is at `0.8.0` now. is it possible to upgrade `marked` version in package.json?

Any word on this?

**Comment by christianalfoni** [comment]
Hi there!

Version `0.8.0` breaks a lot of tests and needs to be reviewed. I have a bit too much on my plate these days, but will look at it if I get a chance! 😄

**Comment by ilias-t** [comment]
any update on this?

**Comment by patsplat** [comment]
https://github.com/storybookjs/marksy/pull/98 should patch this up.  The test failures were b/c of the CI configuration.

**Comment by nikkypyra** [comment]
Could you please update the following package due to vulnerabilities:
marked to 4.0.10 or greater

This will resolve the vulnerability in marked (See [CVE](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21681)).

**Comment by kolesnikanton** [comment]
I updated the marked version here: https://github.com/storybookjs/marksy/pull/367
Could I ask you @Hypnosphi @ndelangen @jimmyandrade to release a new version with the changes?
